core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -133,10 +132,6 @@ class TileImprovement : RulesetStatsObject() {
         val statsDesc = cloneStats().toString()
         if (statsDesc.isNotEmpty()) textList += FormattedLine(statsDesc)
 
-        if (pillageGold > 0) {
-            textList += FormattedLine("Average pillageable gold: ¤[$pillageGold]", link="Tutorial/Pillaging")
```

Comment:
Don't know a cheap solution to get that link back - except [civilopediaText](https://yairm210.github.io/Unciv/Other/Miscellaneous-JSON-files/#civilopedia-text)

Reply:
Maybe by adding an optional link argument for `UniqueType(...)`? For most uniques this argument would be blank and the link would be generated by `getUniqueLink()` in `CivilopediaText.kt` as it's done now but for uniques with non-null link arguments it would use that link instead.

Reply:
A quick and dirty implementation of what I mean.
<Details>

In UniqueTypes.kt
```
enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags: List<UniqueFlag> = emptyList(), val civilopediaLink: String? = null) {

[much later...]

PillageYieldRandom("Pillaging this improvement yields approximately [stats]", UniqueTarget.Improvement, civilopediaLink = "Tutorials/Pillaging"),
```

Then in CivilopediaText.kt

```
private fun getUniqueLink(unique: Unique): String {
            val ruleSet = getCurrentRuleset()
            if (allObjectNamesCategoryMap == null || rulesetCachedInNameMap !== ruleSet)
                allObjectNamesCategoryMap = initNamesCategoryMap(ruleSet)
            if (unique.type?.civilopediaLink != null) return unique.type.civilopediaLink
            for (parameter in unique.params + unique.conditionals.flatMap { it.params }) {
                val category = allObjectNamesCategoryMap!![parameter] ?: continue
                return category.name + "/" + parameter
            }
            return ""
        }
```
</Details>

Reply:
> getUniqueLink()

That was done _long_ before we had the IRulesetObject class hierarchy. Could be done much better nowadays I presume.
Not sure I _can_ like any additional code for links - needs a clear picture which I don't see. Your suggestion _sounds_ a little klugde-y and as if it won't cover all bases, but I may be wrong. The civilopediaText idea at the moment costs nothing except a little json right where the unique itself is, and a revert if we later decide otherwise. And you can give it colors! Any any icon available (if I remember right)!